tests/artifacts/test_artifacts.py

```diff
@@ -224,3 +226,38 @@ def test_artifact_logging_resolution_works_with_non_root_working_directory(text_
             run.info.run_id,
         )
     os.chdir(original_cwd)
+
+
+@pytest.mark.skipif(not is_local_os_windows(), reason="This test only passes on Windows")
+def test_log_artifact_windows_path_with_hostname(text_artifact):
+    experiment_test_1_artifact_location = r"\\my_server\my_path\my_sub_path\1"
```

Comment:
```suggestion
    experiment_test_1_artifact_location = r"\my_server\my_path\my_sub_path\1"
```

Reply:
Yes, just so that on windows, we can force the network drive/server name to be part of the absolute path.

Reply:
Understood :)